core/src/main/resources/lib/layout/task.jelly

```diff
@@ -75,6 +75,9 @@ THE SOFTWARE.
       Generally used with post="true".
       (onclick supersedes this except in the context menu.)
     </st:attribute>
+    <st:attribute name="badge" since="TODO">
+      If set, displays the value as a small badge on the right side of the sidepanel item.
```

Comment:
This needs a better description. At the moment most people will assume this is just a simple text you can pass in but what is actually is required is a jenkins.management.Badge object.